Cubical/Categories/Displayed/Constructions/LeftAdjointToReindex.agda

```diff
@@ -0,0 +1,106 @@
+{-# OPTIONS --safe #-}
+--
+module Cubical.Categories.Displayed.Constructions.LeftAdjointToReindex where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Path
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.Isomorphism
+open import Cubical.Data.Sigma
+
+open import Cubical.Categories.Category.Base
+open import Cubical.Categories.Instances.Terminal
+open import Cubical.Categories.Functor
+
+open import Cubical.Categories.Displayed.Base
+open import Cubical.Categories.Displayed.Properties
+open import Cubical.Categories.Constructions.TotalCategory
+
+private
+  variable
+    ℓC ℓC' ℓCᴰ ℓCᴰ' ℓD ℓD' ℓDᴰ ℓDᴰ' ℓE ℓE' ℓEᴰ ℓEᴰ' : Level
+
+open Categoryᴰ
+
+module _
+  {C : Category ℓC ℓC'} {D : Category ℓD ℓD'}
+  (Cᴰ : Categoryᴰ C ℓDᴰ ℓDᴰ') (F : Functor C D)
+  where
+
+  open Category
+  open Functor F
+  private
+    module Cᴰ = Categoryᴰ Cᴰ
+    open Iso
+
+    hom-path : ∀ {A B A' B'} (p : A ≡ A') (q : B ≡ B') →
+      (D [ A , B ]) ≡ (D [ A' , B' ])
+    hom-path p q = cong₂ (λ a b → D [ a , b ]) p q
+
+    hom-pathP : ∀ {A B A' B'} (p : A ≡ A') (q : B ≡ B') →
+                (f : D [ A , B ]) → (f' : D [ A' , B' ]) →
+                Type ℓD'
+    hom-pathP p q f f' = PathP (λ i → hom-path p q i) f f'
+
+    isPropHomP : ∀ {A B A' B'} (p : A ≡ A') (q : B ≡ B') →
+                (f : D [ A , B ]) → (f' : D [ A' , B' ]) →
+                isProp (hom-pathP p q f f')
+    isPropHomP pdom pcod f f'' x y =
+      isoFunInjective (PathPIsoPath _ _ _) x y fromPathPeq
+      where
+      fromPathPeq : fromPathP x ≡ fromPathP y
+      fromPathPeq = D .isSetHom _ _ (fromPathP x) (fromPathP y)
+
+  ∃F : Categoryᴰ D (ℓ-max (ℓ-max ℓC ℓD) ℓDᴰ) (ℓ-max (ℓ-max ℓC' ℓD') ℓDᴰ')
```

Comment:
Maybe this would be a good place for a comment which says that ```∃F``` is adjoint to reindexing (and just to be sure to say what reindexing means).

Reply:
Does this sound alright?
```
  -- Reindexing a dislayed category Dᴰ over D along a functor F : C → D
  -- gives a displayed category over C
  -- Fiberwise pullback the objects over D along F to display them over C
  --
  --    reindex Dᴰ F                Dᴰ
  --         _                      _
  --         |                      |
  --         v           F          v
  --         C -------------------> D
  --
  -- Which may be read as a 2-functor from displayed categories over D to
  -- displayed categories over C. This operation has a left 2-adjoint, which
  -- we call ∃F, that maps displays over C to displays over D
  --
  --         Cᴰ                  ∃F Cᴰ F
  --         _                      _
  --         |                      |
  --         v           F          v
  --         C -------------------> D
  --
```

Reply:
typo: `dislayed`

Reply:
Very good!